be/test/common/schema_util_test.cpp

```diff
@@ -48,6 +48,18 @@ void construct_subcolumn(TabletSchemaSPtr schema, const FieldType& type, int32_t
     vectorized::PathInData col_path(path);
     subcol.set_path_info(col_path);
     subcol.set_name(col_path.get_path());
+
+    if (type == FieldType::OLAP_FIELD_TYPE_ARRAY) {
+        TabletColumn array_item_col;
+        // double not support inverted index
+        array_item_col.set_type(FieldType::OLAP_FIELD_TYPE_DOUBLE);
```

Comment:
Is it a trick to add a default type DOUBLE that does not support inverted index?

Reply:
Where do you set the real item type and why not set the real item type here?

Reply:
it's ut